lib/active_merchant/billing/compatibility.rb

```diff
@@ -90,8 +90,8 @@ def add_to_base(error)
             add(:base, error)
           end
 
-          def each_full(&block)
-            full_messages.each(&block)
+          def each_full(&)
```

Comment:
Here is one place for what I referred to in the rule above. Here is a link to that rule if the docs help. https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/BlockForwarding